tests/standalone/write-error-001.phpt

```diff
@@ -19,11 +19,11 @@ $wtimeout = 1000;
 $writeConcern = new \MongoDB\Driver\WriteConcern($w, $wtimeout);
 throws(function() use($bulk, $writeConcern, $manager) {
     $result = $manager->executeBulkWrite(NS, $bulk, $writeConcern);
-}, "MongoDB\Driver\Exception\ConnectionException");
+}, "MongoDB\Driver\Exception\BulkWriteException");
 
 ?>
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-OK: Got MongoDB\Driver\Exception\ConnectionException
```

Comment:
Can you add the exception message to this test? Should entail changing `throws(...)` to `echo throws(...), "\n";`. I expect the error is due to using `w` >= 1 on a standalone.